src/collective/cover/tiles/configuration_widgets/cssclasswidget.py

```diff
@@ -21,6 +21,21 @@ def update(self):
         if isinstance(self.context.get('css_class'), unicode):
             self.value = [self.context.get('css_class')]
 
+    def options(self):
+        items = [
+            {
+                key: value
+                for key, value in item.iteritems()
```

Comment:
`iteritems()` works on Python 2 only; you need to import it from `six`; check http://python-future.org/compatible_idioms.html#iterating-through-dict-keys-values-items